connector/servicegraphconnector/connector_test.go

```diff
@@ -6,6 +6,8 @@ package servicegraphconnector
 import (
 	"context"
 	"crypto/rand"
+	"github.com/open-telemetry/opentelemetry-collector-contrib/connector/servicegraphconnector/internal/store"
```

Comment:
This import is in the wrong spot, it should be part third part imports, not within the first part imports.